### PR TITLE
DOCS-2432: Add Agent cloud logging

### PR DESCRIPTION
**Pull request by andf-viam**
Add brief mention to provisioning reference page that Agent logs are now sent to the Viam app > **LOGS** tab.

### CODE REVIEW

File: docs/fleet/provision.md [hunk 1]
```diff
@@ -135,6 +135,16 @@ The Viam Agent is installed as a `systemd` service named `viam-agent`.
 
   For more information, see [Viam Agent management](https://github.com/viamrobotics/agent#management).
 
+### View Viam Agent logs
+
+The Viam Agent writes log messages to the [Viam app](https://app.viam.com/).
+These messages can be found on the [**LOGS** tab](/cloud/machines/#logs) of your machine's page.
```
Comment (JessamyT) [inline docs/fleet/provision.md:141]:
```suggestion
You can find these messages on the [**LOGS** tab](/cloud/machines/#logs) of your machine's page.
```
[opt] passive voice

File: docs/fleet/provision.md [hunk 2]
```diff
@@ -135,6 +135,16 @@ The Viam Agent is installed as a `systemd` service named `viam-agent`.
...
+
+Log messages are only sent when your machine is online and connected to the internet.
```
Comment (JessamyT) [inline docs/fleet/provision.md:143]:
```suggestion
Viam Agent only sends messages when your machine is online and connected to the internet.
```
[opt] passive voice
I'm not actually sure of accuracy of my suggestion, but would be great to say what is sending the messages

File: docs/fleet/provision.md [hunk 3]
```diff
@@ -135,6 +135,16 @@ The Viam Agent is installed as a `systemd` service named `viam-agent`.
...
+If your machine is offline, log messages are queued, and are sent to the Viam app once your machine reconnects to the internet.
+
+These log messages include when `viam-server` is stopped and started, the status of agent subsystems, and any errors or warning encountered during operation.
```
Comment (JessamyT) [inline docs/fleet/provision.md:146]:
```suggestion
These log messages include logs of when `viam-server` is stopped and started, the status of agent subsystems, and any errors or warning encountered during operation.
```
[opt] I don't love my suggestion, but existing wording is grammatically iffy/awk

**Review by JessamyT — APPROVED** [review]
LGTM % passive voice

**Review by ale7714 — APPROVED** [review]
Thanks for the quick update!

File: docs/fleet/provision.md (hunk 3 above)
Comment (andf-viam) [reply docs/fleet/provision.md:146]: This one is too many `logs`

**Pull request merged**